### PR TITLE
[DJ-43] Support

### PR DESCRIPTION
**Pull request by akkt11**

### CODE REVIEW

File: src/shared/services/mutations/index.ts
Comment (ilias-kadyrkulov) [inline src/shared/services/mutations/index.ts:1]: Hey, I thought mutations folder was for all requests except GET?

**Comment by akkt11** [comment]
hi

File: src/shared/services/mutations/index.ts
Comment (akkt11) [reply src/shared/services/mutations/index.ts:1]: Then, i resolved it

Comment (ilias-kadyrkulov) [reply src/shared/services/mutations/index.ts:1]: Yeah, gotcha

**Review by ilias-kadyrkulov — DISMISSED** [review]

**Pull request merged**